client/components/domains/register-domain-step/index.jsx

```diff
@@ -837,13 +836,7 @@ class RegisterDomainStep extends React.Component {
 		const onAddMapping = domain => this.props.onAddMapping( domain, this.state );
 
 		const searchResults = this.state.searchResults || [];
-		const testGroup = abtest( 'domainSuggestionKrakenV313' );
-		let suggestions = includes(
-			[ 'group_1', 'group_2', 'group_3', 'group_4', 'group_5' ],
-			testGroup
-		)
-			? [ ...searchResults ]
-			: reject( searchResults, matchesSearchedDomain );
+		let suggestions = [ ...searchResults ];
```

Comment:
I know you're reverting the patch, but why exactly are we doing this? 
```
let a = [ 1, 2, 3 ];
let b = [ ...a ];
console.log( JSON.stringify( a ) == JSON.stringify( b ) ); // true
```

Reply:
Oh - it's because we're modifying the suggestions in the next few lines and we don't want to modify the prop value :) Initially I tried to fix it but got in an endless loop. Not sure if it's the best way to do it, but since it was like that I guess it's good enough.

Reply:
I see, you just want to copy the value. :+1: